gradle.properties

```diff
@@ -1,7 +1,7 @@
 org.gradle.caching=true
 group=org.ballerinalang
 # During the release workflow, the following will get bumped automatically
-version=2201.2.0-SNAPSHOT
+version=2201.2.0
```

Comment:
Shall we undo the changes in this file and include just the changes to the out files in this PR?

Reply:
Yes sure, this change in this file was added by ballerina-bot. I am relatively new to this, could you please undo the changes or tell me what I have to do to undo changes in this file?

Reply:
You could send a separate PR with just this change. Would that work? You can use the [current master branch of this repo](https://github.com/ballerina-platform/ballerina-distribution) when creating the new branch.